samples/python/classification_sample_async/classification_sample_async.py

```diff
@@ -119,45 +136,29 @@ def main():
     # Create a list to control a order of output
     output_queue = list(range(num_of_input))
 
-    while True:
-        for i in output_queue:
-            # Immediately returns a inference status without blocking or interrupting
-            infer_status = exec_net.requests[i].wait(0)
-
-            if infer_status == StatusCode.RESULT_NOT_READY:
-                continue
-
-            log.info(f'Infer request {i} returned {infer_status}')
-
-            if infer_status != StatusCode.OK:
-                return -2
-
-            # Read infer request results from buffer
-            res = exec_net.requests[i].output_blobs[out_blob].buffer
-            # Change a shape of a numpy.ndarray with results to get another one with one dimension
-            probs = res.reshape(num_of_classes)
-            # Get an array of args.number_top class IDs in descending order of probability
-            top_n_idexes = np.argsort(probs)[-args.number_top :][::-1]
-
-            header = 'classid probability'
-            header = header + ' label' if args.labels else header
-
-            log.info(f'Image path: {args.input[i]}')
-            log.info(f'Top {args.number_top} results: ')
-            log.info(header)
-            log.info('-' * len(header))
-
-            for class_id in top_n_idexes:
-                probability_indent = ' ' * (len('classid') - len(str(class_id)) + 1)
-                label_indent = ' ' * (len('probability') - 8) if args.labels else ''
-                label = labels[class_id] if args.labels else ''
-                log.info(f'{class_id}{probability_indent}{probs[class_id]:.7f}{label_indent}{label}')
-            log.info('')
-
-            output_queue.remove(i)
+    for i in output_queue:
+            
+        res = np.array(list(results[output_queue[i]][1].values()))
```

Comment:
```suggestion
        res = results[i]
```

Reply:
BTW, you can use range directly in loop, without necessary to create output queue variable